main.go

```diff
@@ -64,6 +65,12 @@ func main() {
 		log.Fatal(err)
 	}
 
+	go func(w *watcher.FSWatcher) {
+		for range w.Event {
+			ex.Send(rebuild, nil)
```

Comment:
So this is kind of what I was talking about regarding not needing the exchange here. You could just call the build method directly.